pkg/webui/console/components/device-import-form/fallback-version-ids-section.js

```diff
@@ -0,0 +1,140 @@
+import classnames from 'classnames'
+
+import { useFormContext } from '@ttn-lw/components/form'
+
+import {
+  hasSelectedDeviceRepositoryOther,
+  isOtherOption,
+} from '@console/containers/device-onboarding-form/utils'
+import OtherHint from '@console/containers/device-onboarding-form/type-form-section/repository-form-section/hints/other-hint'
+import BrandSelect from '@console/containers/device-onboarding-form/type-form-section/repository-form-section/device-selection/brand-select'
+import ModelSelect from '@console/containers/device-onboarding-form/type-form-section/repository-form-section/device-selection/model-select'
+import HardwareVersionSelect from '@console/containers/device-onboarding-form/type-form-section/repository-form-section/device-selection/hw-version-select'
+import FirmwareVersionSelect from '@console/containers/device-onboarding-form/type-form-section/repository-form-section/device-selection/fw-version-select'
+import BandSelect from '@console/containers/device-onboarding-form/type-form-section/repository-form-section/device-selection/band-select'
+import style from '@console/containers/device-onboarding-form/type-form-section/repository-form-section/repository.styl'
+
+import tooltipIds from '@ttn-lw/lib/constants/tooltip-ids'
+
+const brandValueSetter = ({ setValues }, { value }) =>
+  setValues(values => ({
+    ...values,
+    version_ids: {
+      ...initialValues.version_ids,
+      brand_id: value,
+    },
+  }))
+const modelValueSetter = ({ setValues }, { value }) =>
+  setValues(values => ({
+    ...values,
+    version_ids: {
+      ...initialValues.version_ids,
+      brand_id: values.version_ids.brand_id,
+      model_id: value,
+    },
+  }))
+
+const initialValues = {
+  version_ids: {
+    brand_id: '',
+    model_id: '',
+    hardware_version: '',
+    firmware_version: '',
+    band_id: '',
+  },
+}
+
+const FallbackVersionIdsSection = () => {
+  const { values } = useFormContext()
+  const { version_ids } = values
+
+  const version = version_ids
+  const brand = version_ids?.brand_id
+  const model = version_ids?.model_id
+  const hardwareVersion = version_ids?.hardware_version
+  const firmwareVersion = version_ids?.firmware_version
+
+  const hasBrand = Boolean(brand) && !isOtherOption(brand)
+  const hasModel = Boolean(model) && !isOtherOption(model)
+  const hasHwVersion = Boolean(hardwareVersion) && !isOtherOption(hardwareVersion)
+  const hasFwVersion = Boolean(firmwareVersion) && !isOtherOption(firmwareVersion)
+
+  const hasSelectedOther = hasSelectedDeviceRepositoryOther(version)
+  const showOtherHint = hasSelectedOther
+
+  return (
+    <Row>
+      <Col>
+        <div className={style.configurationSection}>
+          <BrandSelect
+            className={classnames(style.select, style.selectS)}
+            name="version_ids.brand_id"
+            required
+            tooltipId={tooltipIds.DEVICE_BRAND}
+            valueSetter={brandValueSetter}
```

Comment:
See https://github.com/TheThingsNetwork/lorawan-stack/pull/5823 I have removed the brand request from the useEffect and replaced it with a require request as the was causing flaky tests. We should do the same here wrapping the <FallbackVersionIdsSection /> component so that it requested when you select the inputMethod